transaction/src/plan/action/output.rs

```diff
@@ -61,6 +61,19 @@ impl OutputPlan {
         }
     }
 
+    /// Convenience async method to construct the [`Output`] described by this
+    /// [`OutputPlan`].
+    pub async fn output_concurrent(
+        &self,
+        ovk: &OutgoingViewingKey,
+        memo_key: &PayloadKey,
+    ) -> Output {
+        Output {
+            body: self.output_body(ovk, memo_key),
+            proof: self.output_proof(),
+        }
+    }
+
```

Comment:
what's the reason to have a duplicate method here, rather than calling the existing method in an async context?